client/client.go

```diff
@@ -477,6 +487,25 @@ type (
 		// API. If the check fails, an error is returned.
 		CheckHealth(ctx context.Context, request *CheckHealthRequest) (*CheckHealthResponse, error)
 
+		// UpdateWorkflow issues an update request to the specified
+		// workflow execution and returns the result synchronously. Calling this
+		// function is equivalent to calling UpdateWorkflowOptions with
+		// the same arguments and indicating that the RPC call should wait for
+		// completion of the update process.
+		// NOTE: Experimental
```

Comment:
```suggestion
		//
		// WARNING!! This API is experimental and subject to incompatible changes.
```

Wonder if we want to make this a little louder.

Reply:
Hrmm, I guess if there's so many we should at least do it with the entry points. Probably weren't loud enough there. But this is fine too, no need to change anything.